model/main.py

```diff
@@ -0,0 +1,27 @@
+from model.model import Model
+
+from argparse import ArgumentParser, Namespace
+
+
+def get_args() -> Namespace:
+    parser = ArgumentParser(description="Read file from Command line.")
+    parser.add_argument(
+        "--aug-airports-load-file",
+        required=True,
+        type=str,
+        help="file path for information about each airport",
+    )
+    parser.add_argument(
+        "--model_dump_dir",
+        required=True,
+        type=str,
+        help="path to directory to store model",
+    )
+    return parser.parse_args()
+
+
+if __name__ == "__main__":
+    args = get_args()
```

Comment:
I don't think this will work as you're not giving it anything to parse? If you update the method as suggested above this should look something more like

```python
parser = setup_parser()
args = parser.parse_args(sys.argv[1:])
```